client/me/developer/features/handle-click-link.tsx

```diff
@@ -9,5 +14,7 @@ export const handleClickLink = ( event: React.MouseEvent< HTMLAnchorElement > )
 		featureSlug = featureSlug.substring( pathIndex + prefixToRemove.length );
 	}
 
-	recordTracksEvent( 'calypso_me_developer_learn_more', { feature: featureSlug } );
+	const recordEvent = withEnhancers( recordTracksEvent, enhanceWithUserIsDevAccount );
+
+	dispatch( recordEvent( 'calypso_me_developer_learn_more', { feature: featureSlug } ) );
```

Comment:
```suggestion
	recordEvent( 'calypso_me_developer_learn_more', { feature: featureSlug } )( dispatch, getState, … );
```
Given that we don't have access to `getState` here, I wonder if we shouldn't consider not reusing `enhanceWithUserIsDevAccount` here after all

Reply:
😓 I just came to realize that we're also calling `handleClickLink` here: https://github.com/Automattic/wp-calypso/blob/516d51b6d5aca9995a841f912c45bf3c3e092b6a/client/me/developer/features/use-features-list.tsx#L83-L107

It's a higher priority to get page view tracking out. I'm going to revert the link tracking stuff and only enhance the page view events! We can followup with a separate PR to address click tracking!